src/Azure.Functions.Cli/Helpers/AzureHelper.cs

```diff
@@ -62,7 +66,15 @@ private static async Task<Site> TryGetFunctionAppFromArg(string name, IEnumerabl
             {
                 string siteId = await GetResourceIDFromArg(subscriptions, query, accessToken, managementURL);
                 var app = new Site(siteId);
-                await LoadFunctionApp(app, accessToken, managementURL);
+                if (loadFunction != null)
+                {
+                    await loadFunction(app, accessToken, managementURL);
```

Comment:
Could you add a comment here explaining the difference between `loadFunction` and `LoadFunctionApp`? While the usage here is straightforward, I think it would be good to clarify when to use each method. Perhaps we could also rename the parameter `loadFunction` as `loadContainerApp`. In general, however, if another service would like to add support on top of Core Tools, adding an entirely new `loadFunction` parameter to this method doesn't appear to be sustainable. Is there a way we could pass an enumerable here instead to describe which type of site we are trying to load?

Reply:
Added the comment.